### PR TITLE
FIX PROJECT + DEFAULT OPEN on Accordion

### PR DESCRIPTION
**Pull request by pierry01**
### FIX PROJECT BECAUSE NAME CHANGED FROM RBUI TO RUBY_UI
image

### ACCORDION VIEW WITH DEFAULT OPEN
image

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -75,7 +75,7 @@ end
 
 gem "phlex-rails"
 
-gem "rbui", github: "rbui-labs/rbui", branch: "main"
-# gem "rbui", path: "../rbui"
+gem "ruby_ui", github: "rbui-labs/ruby_ui", branch: "main"
+# gem "ruby_ui", path: "../ruby_ui"
```
Comment (pierry01) [inline Gemfile:79]: there is still a lot of typo that rbui should be ruby_ui, but it deserves another PR

**Comment by cirdes** [comment]
@pierry01, let's wait for @SethHorsley to rename it first
